section4/exercises/burrito.rb

```diff
@@ -1,19 +1,39 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
-end
+
+  def add_topping(toppings)
+    @toppings += [toppings]
+  end
+
```

Comment:
Here is an example of containing your methods within your class. For these methods, you are adding, or subtracting, to an array, not a number, so you will need to use methods rather than calculation. Nice job using arguments with these methods!